openbas-model/src/main/java/io/openbas/database/model/Executable.java

```diff
@@ -26,6 +28,13 @@ public class Executable extends Payload {
   @JsonProperty("executable_file")
   private Document executableFile;
 
+  @Queryable(filterable = true, searchable = true)
+  @Column(name = "executable_arch")
+  @JsonProperty("executable_arch")
+  @Enumerated(EnumType.STRING)
+  @NotNull
+  private Endpoint.PLATFORM_ARCH executableArch;
+
```

Comment:
Maybe extract this Platform_arch enum in other class?

Reply:
I'm merging without this refacto